tests/unit/lms/views/dashboard/api/user_test.py

```diff
@@ -38,6 +38,7 @@ def test_get_students(self, user_service, pyramid_request, views, get_page):
             course_ids=sentinel.course_ids,
             assignment_ids=sentinel.assignment_ids,
             segment_authority_provided_ids=sentinel.segment_authority_provided_ids,
+            h_userids=None,
```

Comment:
The only tweak to the tests, filtering by users is not used on the user dropdown (that's a toughts sentence to read) but we pass the `None` parameter now because we use all the parameters as we share the helper method.